lib/plugin/selenoid.js

```diff
@@ -0,0 +1,206 @@
+const util = require('util');
+const axios = require('axios').default;
+const exec = util.promisify(require('child_process').exec);
+const {
+  container, event, recorder, output,
+} = require('../index');
+
+const dockerCreateScriptArr = [
+  'docker create --rm --name $name$',
+  '-p $port$:$port$',
```

Comment:
I think it's not correct, inside a container a port will always be 4444 but for user (on host machine) this can be different